atomix/cluster/src/test/java/io/atomix/raft/storage/system/MetaStoreTest.java

```diff
@@ -216,4 +216,26 @@ public void shouldLoadLatestConfigurationAfterRestart() throws IOException {
     assertThat(readConfig.members())
         .containsExactlyInAnyOrder(secondConfig.members().toArray(new RaftMember[0]));
   }
+
+  @Test
+  public void shouldStoreAndLastFlushedIndex() {
+    // given
+    metaStore.storeFlushedIndex(5L);
+
+    // when/then
+    assertThat(metaStore.loadFlushedIndex()).isEqualTo(5L);
+  }
+
+  @Test
+  public void shouldStoreAndLoadLastFlushedIndexAfterRestart() throws IOException {
+    // given
+    metaStore.storeFlushedIndex(5L);
```

Comment:
Please also add a test where you update the flushedIndex multiple times and can read the latest.